src/main/kotlin/org/rust/cargo/runconfig/customBuild/CustomBuildConfigurationEditor.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.runconfig.customBuild
+
+import com.intellij.openapi.fileChooser.FileChooserDescriptorFactory
+import com.intellij.openapi.options.SettingsEditor
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.TextFieldWithBrowseButton
+import com.intellij.ui.components.CheckBox
+import com.intellij.ui.dsl.builder.RowLayout
+import com.intellij.ui.dsl.builder.panel
+import org.rust.RsBundle
+import org.rust.openapiext.fullWidthCell
+import org.rust.stdext.toPath
+import javax.swing.JCheckBox
+import javax.swing.JComponent
+
+class CustomBuildConfigurationEditor(val project: Project)
```

Comment:
Shouldn't the editor somehow show a (non-editable) path to the crate root in order to distinguish buildscript run configurations from each other?